serverless/pages/manage-org.asciidoc

```diff
@@ -1,15 +1,149 @@
+
+For example, you can assign a user the developer role for a specific {es-serverless} project:
+
+[role="screenshot"]
+image::images/individual-role.png[Individual role]
+
+ifdef::serverlessCustomRoles[]
+
+You can also optionally <<custom-roles,create custom roles in a project>>.
+To assign a custom role to users, go to "Instance access roles" and select it from the list under the specific project it was created in.
+
+endif::[]
+
+[discrete]
+[[general-assign-user-roles-table]]
+|===
+|Name  |Description  |Available
+
+|Admin |Has full access to project management, properties, and security privileges. Admins log into projects with superuser role privileges. |{es-badge}{obs-badge}{sec-badge}
+
+|Developer |Creates API keys, indices, data streams, adds connectors, and builds visualizations. |{es-badge}
+
+|Viewer |Has read-only access to project details, data, and features. |{es-badge}{obs-badge}{sec-badge}
+
+|Editor |Configures all Observability or Security projects. Has read-only access to data indices. Has full access to all project features. |{obs-badge}{sec-badge}
+
+|Tier 1 analyst |Ideal for initial alert triage. General read access, can create dashboards and visualizations. |{sec-badge}
+
+|Tier 2 analyst |Ideal for alert triage and beginning the investigation process. Can create cases. |{sec-badge}
+
+|Tier 3 analyst |Deeper investigation capabilities. Access to rules, lists, cases, Osquery, and response actions. |{sec-badge}
+
+|Threat intelligence analyst |Access to alerts, investigation tools, and intelligence pages. |{sec-badge}
+
+|Rule author |Access to detection engineering and rule creation. Can create rules from available data sources and add exceptions to reduce false positives. |{sec-badge}
+
+|SOC manager |Access to alerts, cases, investigation tools, endpoint policy management, and response actions. |{sec-badge}
+
+|Endpoint operations analyst |Access to endpoint response actions. Can manage endpoint policies, {fleet}, and integrations. |{sec-badge}
+
+|Platform engineer |Access to {fleet}, integrations, endpoints, and detection content. |{sec-badge}
+
+|Detections admin |All available detection engine permissions to include creating rule actions, such as notifications to third-party systems. |{sec-badge}
+
+|Endpoint policy manager |Access to endpoint policy management and related artifacts. Can manage {fleet} and integrations. |{sec-badge}
+|===
+
+
+[discrete]
+[[general-leave-an-organization]]
+== Leave an organization
+
+On the **Organization** page, click **Leave organization**.
+
+If you're the only user in the organization, you can only leave if you have deleted all your projects and don't have any pending bills.
+
+[discrete]
+[[general-join-organization-from-existing-cloud-account]]
+== Join an organization from an existing Elastic Cloud account
+
+// :description: Join a new organization and bring over your projects.
```

Comment:
Why does this have bring over your projects when you can't bring over your projects?

Reply:
does this mean that you cannot be a member of more than one organization? I'm assuming this is email driven (and probably therefore applies to SSO)?

Reply:
I'm also assuming this applies generally (deployments and serverless) and is an org thing and not a serverless only thing.

Reply:
This line seems to be commented out so not taken into account.
You're correct on both assumptions: For now a user can only belong to 1 organization. If they want to join another, they either need to leave the first one, or to use a different email. The "join from existing account" section seems like it's been copied over from the cloud docs. It's not specific to just serverless. I'm not sure it's relevant to keep here.

Reply:
cleared up the verbiage to be more clear